From 75c57b5f810d6ac6f393459d2884877c7fcf774c Mon Sep 17 00:00:00 2001
From: Bruno Pitrus <brunopitrus@hotmail.com>
Date: Mon, 05 Jun 2023 21:22:55 +0000
Subject: [PATCH] IWYU: Add missing standard includes for GCC13

Bug: 957519
Change-Id: Ie327653083173b41dfd5a456f82ccb490c305c26
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/4547709
Commit-Queue: Nico Weber <thakis@chromium.org>
Auto-Submit: Bruno Pitrus <brunopitrus@hotmail.com>
Reviewed-by: Nico Weber <thakis@chromium.org>
Reviewed-by: Eric Orth <ericorth@chromium.org>
Cr-Commit-Position: refs/heads/main@{#1153468}
---

diff --git a/base/cpu.h b/base/cpu.h
index 10450e9..2ac00e9 100644
--- a/base/cpu.h
+++ b/base/cpu.h
@@ -5,6 +5,7 @@
 #ifndef BASE_CPU_H_
 #define BASE_CPU_H_
 
+#include <cstdint>
 #include <string>
 
 #include "base/base_export.h"
diff --git a/components/services/app_service/public/cpp/intent_filter.h b/components/services/app_service/public/cpp/intent_filter.h
index 9c83ccf..48724963 100644
--- a/components/services/app_service/public/cpp/intent_filter.h
+++ b/components/services/app_service/public/cpp/intent_filter.h
@@ -5,6 +5,7 @@
 #ifndef COMPONENTS_SERVICES_APP_SERVICE_PUBLIC_CPP_INTENT_FILTER_H_
 #define COMPONENTS_SERVICES_APP_SERVICE_PUBLIC_CPP_INTENT_FILTER_H_
 
+#include <memory>
 #include <set>
 #include <string>
 #include <utility>
diff --git a/components/services/app_service/public/cpp/permission.h b/components/services/app_service/public/cpp/permission.h
index 82d323a..ede8540 100644
--- a/components/services/app_service/public/cpp/permission.h
+++ b/components/services/app_service/public/cpp/permission.h
@@ -5,6 +5,7 @@
 #ifndef COMPONENTS_SERVICES_APP_SERVICE_PUBLIC_CPP_PERMISSION_H_
 #define COMPONENTS_SERVICES_APP_SERVICE_PUBLIC_CPP_PERMISSION_H_
 
+#include <memory>
 #include <utility>
 #include <vector>
 
diff --git a/components/viz/common/view_transition_element_resource_id.h b/components/viz/common/view_transition_element_resource_id.h
index 17ae568..44f980f 100644
--- a/components/viz/common/view_transition_element_resource_id.h
+++ b/components/viz/common/view_transition_element_resource_id.h
@@ -7,6 +7,7 @@
 
 #include <stdint.h>
 
+#include <cstdint>
 #include <string>
 #include <vector>
 
diff --git a/net/cert/pki/string_util.h b/net/cert/pki/string_util.h
index b18f716..7340408 100644
--- a/net/cert/pki/string_util.h
+++ b/net/cert/pki/string_util.h
@@ -9,6 +9,7 @@
 
 #include <stdint.h>
 
+#include <cstdint>
 #include <string_view>
 #include <vector>
 
